interface/main/calendar/add_edit_event.php

```diff
@@ -345,7 +345,7 @@ function setEventDate($start_date, $recurrence)
 
              // the starting date of the event, pay attention with this value
```

Comment:
don't lines 559 and 563 need to be dealt with?

Reply:
and lines 678 and 681

Reply:
also line 195

Reply:
Also should look into $_POST['selected_date'] and $_POST['event_start_date'] to see if any work is needed on those.

Reply:
selected_date and event_start_date are hidden fields so I did not touch them (they are always in yyyy-mm-dd format) .

Reply:
Then can the fixDate calls be removed?

Reply:
Still there are a lot of pages that use this function.

Reply:
But if selected_date and event_start_date are hidden fields and always yyyy-mm-dd, won't fixDate break them?

Reply:
O.K right, removed from this page.